src/core/Tw2Mesh.js

```diff
@@ -77,12 +83,21 @@ function Tw2Mesh()
     this.geometryResPath = '';
     this.lowDetailGeometryResPath = '';
     this.geometryResource = null;
+
     this.opaqueAreas = [];
     this.transparentAreas = [];
     this.additiveAreas = [];
     this.pickableAreas = [];
     this.decalAreas = [];
     this.depthAreas = [];
+
+    this.display = true;
```

Comment:
Not sure what you mean sorry, I've never used the debugIsHidden (or any debugging for that matter) so haven't touched it.